src/types.ts

```diff
@@ -35,4 +38,19 @@ export type ReducerActions =
 export interface PlotChildren {
   invalidChild: boolean;
   lineSeries: ReactElement<LineSeriesProps>[];
+  axis: Record<'x' | 'y', ReactElement<AxisProps> | null>;
+}
+
+export interface AxisProps {
+  fontSize?: number;
+  label?: string;
+  showTicks?: boolean;
```

Comment:
can we call it `showGridLines` instead ? the ticks are the little bars coming out of the axes

Reply:
I was also thinking about the name, I followed the D3 suggestion, but I prefer your idea